examples/notebooks/03_model_scoring.py

```diff
@@ -201,23 +162,17 @@
 
 # %%
 import matplotlib.pyplot as plt
+from earth2mip.forecast_metrics_io import read_metrics
+import pandas as pd
 
-dataset = xarray.open_dataset("scoring_output.nc")
+series = read_metrics("scoring_output")
+dataset = time_average_metrics(series)
 
 plt.close("all")
 fig, ax = plt.subplots(figsize=(10, 5))
-ax.plot(dataset.rmse.sel(channel="z500").values)
-
-# Plot clean up
-ticks = np.arange(0, 41, 5)
-ax.set_xticks(ticks)
-ax.set_xticklabels([f"{6*i}" for i in ticks])
-ax.set_xlabel("Lead Time")
+t = dataset.lead_time / pd.Timedelta("1 h")
```

Comment:
much cleaner thanks!

Reply:
:)

Reply:
to bad matplotlib doesn't support timedeltas better. I think the default is nano-seconds.